backend/src/main/java/com/cruru/applicant/service/ApplicantCardFilter.java

```diff
@@ -0,0 +1,29 @@
+package com.cruru.applicant.service;
+
+import com.cruru.applicant.domain.EvaluationStatus;
+import com.cruru.applicant.domain.dto.ApplicantCard;
+
+public class ApplicantCardFilter {
+
+    private ApplicantCardFilter() {}
+
+    public static boolean filterByScore(ApplicantCard card, Double minScore, Double maxScore) {
+        double avgScore = card.averageScore();
+        return avgScore >= minScore && avgScore <= maxScore;
```

Comment:
이렇게 표현하면 가독성이 더 좋을 것 같네요!

```suggestion
        return minScore <= avgScore && avgScore <= maxScore;
```